### ISSUE TITLE
Webpage Update (Indian Fund Code)

### ISSUE THREAD

**Issue by merrellc** [ONRR/onrr.gov-site#452]
Please update the following webpage http://onrr.gov/reportpay/payments.htm with the attached files:
Indian Leases Cross-walked to Distributee Codes
Downloadable Excel Version Indian Leases Cross-walked to Distributee Codes

Also update to read 01/15/21 for both files.

In addition, please update to 01/15/21 on URL http://www.onrr.gov/ReportPay/royalty-reporting.htm for the same files linked to this URL.
[FCIL.pdf](https://github.com/ONRR/onrr.gov-site/files/5823256/FCIL.pdf)
[FCIL.xlsx](https://github.com/ONRR/onrr.gov-site/files/5823260/FCIL.xlsx)

**Comment by lpgoldstein** [comment]
@merrellc I've attached the accessibility report for the pdf. Somehow this pdf has been saved as a form so there is an error for that. The report also shows that there are no bookmarks. The Excel file looks great. Thanks! 
[FCIL.pdf Accessibility Report.docx](https://github.com/ONRR/onrr.gov-site/files/5835757/FCIL.pdf.Accessibility.Report.docx)

**Comment by merrellc** [comment]
See attached
[FCIL.pdf](https://github.com/ONRR/onrr.gov-site/files/5838990/FCIL.pdf)

**Comment by lpgoldstein** [comment]
@merrellc Thanks! The only thing it is missing now are bookmarks.

**Comment by merrellc** [comment]
[FCIL.pdf](https://github.com/ONRR/onrr.gov-site/files/5845388/FCIL.pdf)
See update, Thank you

**Comment by lpgoldstein** [comment]
@merrellc Looks great, thanks! Please review: http://2017onrr/ReportPay/payments.htm
http://2017onrr/ReportPay/royalty-reporting.htm

**Comment by merrellc** [comment]
Ready to post. Thank you


Kindly,
​Candace Merrell
Office of Natural Resources Revenue
Reference and Reporting Management
Candace.Merrell@onrr.gov
303-231-3517

Warning:  This message is intended only for use of the individual or entity to which it is addressed and may contain information that is privileged or confidential and exempt from disclosure under applicable law.  If the reader of this message is not the intended recipient or the employee or agent responsible for delivering this message to the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited.  If you have received this communication in error, please notify the sender immediately by return e-mail.
________________________________

**Comment by lpgoldstein** [comment]
@merrellc changes are live!
